src/mbgl/map/tile_worker.cpp

```diff
@@ -41,14 +41,13 @@ TileParseResult TileWorker::parseAllLayers(std::vector<std::unique_ptr<StyleLaye
                                            PlacementConfig config) {
     // We're doing a fresh parse of the tile, because the underlying data has changed.
     pending.clear();
+    placementPending.clear();
     partialParse = false;
+    placementConfig = config;
```

Comment:
yes, that would be cleaner and it could avoid an extra `redoPlacement` in some cases. changed in 716b01c